src/Data/Nat/Primality/Rough.agda

```diff
@@ -0,0 +1,53 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Natural numbers whose prime factors are all above a bound
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+module Data.Nat.Primality.Rough where
+
+open import Data.Nat.Base using (ℕ; suc; _≤_; _<_; z≤n; s≤s; _+_)
+open import Data.Nat.Divisibility using (_∣_; _∤_; ∣-trans; ∣1⇒≡1)
+open import Data.Nat.Induction using (<-rec; <-Rec)
+open import Data.Nat.Primality using (Prime; composite?)
+open import Data.Nat.Properties using (_≟_; <-trans; ≤∧≢⇒<; m<1+n⇒m<n∨m≡n)
+open import Data.Product.Base using (_,_)
+open import Data.Sum.Base using (inj₁; inj₂)
+open import Function.Base using (_∘_; flip)
+open import Relation.Nullary.Decidable.Core using (yes; no)
+open import Relation.Binary.PropositionalEquality.Core using (refl)
+
+-- A number is k-rough if all its prime factors are greater than or equal to k
```

Comment:
Is this a standard notion? A reference in a comment would be good, if so.

Reply:
It's not something that I made up, at least. The notion has a Wikipedia page, but seems to originate with OEIS contributions. I'm not sure if there's a good reference I can link.